### DISCUSSION TITLE
Grafana lightweight alternatives?

### DISCUSSION THREAD

**Discussion by mzy2240** [ytyou/ticktock#29] [General]
Targeting running in the single board computer(and other edge devices), I’m very happy with TickTock in terms of cpu and memory usage. It is a great replacement of InfluxDB in those low resource environments. So naturally I would like to replace the not-lightweight-anymore grafana as well. Any idea or recommends on this? Of course I could simply write my own web dashboard and find a lightweight web server to host, but is there any out-of-box alternatives?

**Comment by ylin30** [comment]
I did manage to run Grafana in PI before but, yes, Grafana is heavy weight since it includes lots of features, especially if you use MySQL instead of Sqlite as storage for Grafana. Try Sqlite if you used MySQL. 

You may consider other lightweight frontend options like echarts (https://github.com/apache/echarts) and Alibaba's G2. They both include JS packages as needed instead of importing all. I haven't tried them yet.
